src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -17,6 +17,21 @@ universes v u
 @[derive decidable_eq] inductive walking_parallel_pair : Type v
 | zero | one
 
+instance fintype_walking_parallel_pair : fintype walking_parallel_pair :=
+{ elems := [walking_parallel_pair.zero, walking_parallel_pair.one].to_finset,
+  complete := λ x, by { cases x; simp } }
+
+section
+open tactic
+/-- Applies `cases` on a `walking_parallel_pair` hypothesis. -/
+-- TODO find a more general approach!
+meta def cases_walking_parallel_pair : tactic unit :=
```

Comment:
Where do you use this tactic?

Reply:
I guess it's not used in this PR, but certainly when I do things like construct limits from products and equalizers it's useful to be able to case bash. Perhaps I should postpone this for that PR?

Reply:
Yeah, let's add it once it gets used.